Runtime/Scripts/WebRTC.cs

```diff
@@ -230,6 +231,23 @@ public static class WebRTC
 
         public static void Initialize(EncoderType type = EncoderType.Hardware)
         {
+            if (Application.platform != RuntimePlatform.LinuxEditor &&
+                Application.platform != RuntimePlatform.LinuxPlayer)
+            {
+                if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLCore ||
+                    SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES2 ||
+                    SystemInfo.graphicsDeviceType == GraphicsDeviceType.OpenGLES3)
+                {
+                    Debug.LogError($"Not Support OpenGL API on {Application.platform}");
+#if UNITY_EDITOR
+                    UnityEditor.EditorApplication.isPlaying = false;
+                    return;
+#else
+                    throw new NotSupportedException();
```

Comment:
Good, `NotSupportedException` it is correct to use in this situation.
This could be better if you add a comment for telling what happened.

https://docs.microsoft.com/en-us/dotnet/api/system.notsupportedexception.-ctor?view=netframework-4.8#System_NotSupportedException__ctor_System_String_